packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss

```diff
@@ -9,6 +9,8 @@ $body-line-height: calc(64px - 2rem);
   color: #253b5b;
   border-radius: $border-radius;
   box-shadow: 0 1px 2px 0 rgba(000, 000, 000, .15);
+
```

Comment:
ok so.. why the blank line 😛

Reply:
Oops this shouldn't have been committed... nasty!